Formula/tenyr.rb

```diff
@@ -30,18 +30,14 @@ def install
     # JIT build is not available until we can pull in AsmJit somehow
     # HEAD version can build with JIT enabled, using git submodule
     # Right now there is no way for `build.with?("jit")` to be true
-    if build.without?("jit")
-      args << "JIT=0"
-    end
+    args << "JIT=0" if build.without?("jit")
```

Comment:
The `()` are redundant here, although that has never been consistently enforced on formulae.